package.json

```diff
@@ -1,13 +1,15 @@
 {
   "name": "react-calendar-timeline",
-  "version": "0.15.1",
+  "version": "0.16.0",
   "description": "react calendar timeline",
   "main": "lib/index.js",
   "module": "src/index.js",
   "scripts": {
     "build": "npm run build:lib",
     "build:demo": "echo '!!! Building Demo' && BABEL_ENV=react node build.js production",
-    "build:lib": "echo '!!! Building Library' && ./node_modules/.bin/babel src --out-dir lib && ./node_modules/.bin/node-sass src/lib/Timeline.scss lib/lib/Timeline.css && sed -i'.bak' 's/Timeline\\.scss/Timeline\\.css/g' lib/lib/Timeline.js && rm lib/lib/Timeline.js.bak",
+    "build:lib": "echo '!!! Building Library' && npm run babel  && npm run sass && sed -i'.bak' 's/Timeline\\.scss/Timeline\\.css/g' lib/lib/Timeline.js && rm lib/lib/Timeline.js.bak",
+    "babel": "node_modules/.bin/babel src --out-dir lib",
+    "sass": "node_modules/.bin/node-sass src/lib/Timeline.scss lib/lib/Timeline.css",
```

Comment:
Should explain... this enables react-calendar-timeline to be built in Git bash environment in Windows.

Reply:
Hi @KryptosX would you mind removing this change as it's outside the context of this issue? I believe this [issue](https://github.com/namespace-ee/react-calendar-timeline/issues/181) is already noted, though there is no PR for it so you could do that too if you wanted!

Reply:
No problem, will do it now.